source/framework/tools/src/TRestTools.cxx

```diff
@@ -115,14 +115,14 @@ void TRestTools::LoadRESTLibrary(bool silent) {
             }
             const TString pathRootString = it.path().string();
             TString libName = TRestTools::SeparatePathAndName((std::string)pathRootString).second;
-            if (!libName.Contains("Rest")) {
+            if (!libName.Contains("Rest", TString::kExact)) {
                 // e.g. "libRestFramework.so"
                 continue;
             }
             // Check if library is excluded from loading e.g. is from a package
             bool excluded = false;
             for (const TString excludedLibrary : excludedLibraries) {
-                if (libName.Contains(excludedLibrary)) {
+                if (libName.Contains(excludedLibrary, TString::kIgnoreCase)) {
```

Comment:
As explained in the PR text its because the package name is `restG4` but the generated library name contains `RestG4`. In the future instead of hardcoding the package names we may get them automatically so thats why I put the ignore case there.